dfget/core/regist/register.go

```diff
@@ -105,7 +105,7 @@ func (s *supernodeRegister) checkResponse(resp *types.RegisterResponse, e error)
 		return errortypes.New(constants.HTTPError, e.Error())
 	}
 	if resp == nil {
-		return errortypes.New(constants.HTTPError, "empty response, unknown error")
+		return errortypes.New(constants.HTTPError, "empty response, ErrUnknowError")
```

Comment:
Why did you change the error message?

Reply:
Because of your problem with the way you express.

Reply:
I can't understand what you really mean.

Reply:
Or you are not clear about what you said.

Reply:
😢

Reply:
I am sorry.